backend/coreapp/compilers.py

```diff
@@ -747,6 +791,8 @@ def preset_from_name(name: str) -> Optional[Preset]:
         MWCC_30_137,
         "-O4,p -enum int -lang c99 -Cpp_exceptions off -gccext,on -gccinc -interworking -gccdep -MD",
     ),
+    # MACOS9
+    Preset("The Sims 1", MWCPPC_24, "-lang=c++ -O2 -str pool -g"),
```

Comment:
I think The Sims is probably better, since that's the official name. Also, I think you were saying earlier it's -O3?